convert2rhel/pkghandler.py

```diff
@@ -251,7 +251,8 @@ def get_third_party_pkgs():
 
 
 def get_installed_pkgs_w_different_fingerprint(fingerprints, name=""):
-    """Return list of all the packages (rpm objects) that are not signed
+    """Return list of all the packages (yum.rpmsack.RPMInstalledPackage objects in case
+    of yum and hawkey.Package objects in case of dnf) that are not signed
```

Comment:
Unrelated to this PR, but should we have a wrapper around `rpmsack.RPMInstalledPackage` and `hawkey.Package`?

Reply:
Yes. Since this is related to using dnf introduced in https://github.com/oamg/convert2rhel/pull/82, I'm going to leave that to the "_Move yum/dnf-related code to a new module handling package management_" refactoring related to https://github.com/oamg/convert2rhel/pull/82#discussion_r465137862.